app/models/article.rb

```diff
@@ -567,6 +568,17 @@ def set_caches
     self.path = calculated_path.downcase
   end
 
+  def normalize_title
+    return unless title
+
+    self.title = title
```

Comment:
Wondering about leveraging a new method for https://github.com/forem/forem/blob/fbf16f54cef0d47ad07b35d52d6f18dc2c88e4f0/app/lib/string_attribute_cleaner.rb#L1

Like we have nullify but we don't have normalize.  (that could be a second refactor)

Reply:
@jeremyf This is interesting. I wonder if we'd need to use this for things other than post titles. Maybe display names for users, since author names are shown next to post titles in the list?

Reply:
I'm starting working on it (because my "regular" work's done for the week and I'm waiting on a sleeping puppy)